### DISCUSSION TITLE
Long column alias truncated

### DISCUSSION THREAD

**Discussion by dotNetNoob** [dbeaver/dbeaver#20778] [Q&A] [answered]
Hi All: I'm building an extract CSV for a vendor and their requirement is to have some  field names longer than 65 chars. When I export the query results I see some errors as below:

> identifier "Serv_PROV_ADRS_LINE_1_TXTSRC_PROV_ADRS_LINE_1_TXTRPTG_PROV_ADRS_LINE_1_TXT" will be truncated to "Serv_PROV_ADRS_LINE_1_TXTSRC_PROV_ADRS_LINE_1_TXTRPTG_PROV_ADRS"

I asked the vendor if we could truncate the field name and they are insisting that it needs to be exactly as they've specified. Is there a way to increase the length of the field name?
![dbeaverFieldTruncation](https://github.com/dbeaver/dbeaver/assets/5372584/b7246d7b-9841-4ebc-aacc-11aecdc75a39)

**Comment by E1izabeth** [comment]
What database do you use?

**Reply by dotNetNoob** [reply]
Hi Elizabeth: I'm pulling the data from PostGresql, but the column name is being aliased manually during the query, like:

select foo "LONG_COLUMN_NAME" from...

So I'm not sure if it has anything to do with the underlying data source?

**Reply by dotNetNoob** [reply]
This is for DBeaver 23.1.3 running on Windows 10 professional

**Comment by E1izabeth** [comment, marked as answer]
It's a database limitation https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
> By default, NAMEDATALEN is 64 so the maximum identifier length is 63 bytes. If this limit is problematic, it can be raised by changing the NAMEDATALEN constant in src/include/pg_config_manual.h.
